README.md

```diff
@@ -1 +1,62 @@
-# akka-http-java-seed.g8
\ No newline at end of file
+A [Giter8][g8] template for Akka HTTP in Java
+
+Prerequisites:
+- JDK 8
+- sbt 0.13.13 or higher
```

Comment:
add a link to http://www.scala-sbt.org/download.html - java people may not know about it